core/src/main/java/tc/oc/pgm/api/setting/SettingValue.java

```diff
@@ -71,6 +105,25 @@ public String getName() {
     return name;
   }
 
+  /**
+   * Gets the formatted name of the {@link SettingValue}.
+   *
+   * @return formatted name of the {@link SettingValue}.
+   */
+  public Component getDisplayName() {
```

Comment:
`getName`. "display name" and "lore" and just implementation details of the menu.

Reply:
Are you suggesting I move the formatted name and description out of the Settings objects and into the menu implementation?

Reply:
It's alright, just keep it as is.